src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -346,7 +348,8 @@ private function registerSwaggerConfiguration(ContainerBuilder $container, array
 
         $loader->load('swagger.xml');
 
-        if ($config['enable_swagger_ui']) {
+        $bundles = $container->getParameter('kernel.bundles');
+        if ($config['enable_swagger_ui'] && isset($bundles['TwigBundle'])) {
```

Comment:
Maybe should we just throw if `enable_swagger_ui` is `true` but Twig isn't available. It would be easier to debug (the exception's message should recommend to install `twig-bundle`.

Reply:
The default value of `enable_swagger_ui` can be false if twig isn't available btw (in the `Configuration` class). As you'll not be able to access to the installed bundle list in this class, you can fallback on checking if `class_exists(TwigBundle::class)`.